src/cephadm/cephadm.py

```diff
@@ -3071,10 +3075,14 @@ def get_container_mounts(ctx, fsid, daemon_type, daemon_id,
     if daemon_type == 'osd':
         # selinux-policy in the container may not match the host.
         if HostFacts(ctx).selinux_enabled:
-            selinux_folder = '/var/lib/ceph/%s/selinux' % fsid
-            if not os.path.exists(selinux_folder):
-                os.makedirs(selinux_folder, mode=0o755)
-            mounts[selinux_folder] = '/sys/fs/selinux:ro'
+            cluster_dir = f'/var/lib/ceph/{fsid}'
```

Comment:
We should probably use `ctx.data_dir` instead of hardcoding `/var/lib/ceph/` here.